### DISCUSSION TITLE
Vue Router problem

### DISCUSSION THREAD

**Discussion by NotatNiiK** [vuejs/router#1965] [Help and Questions] [answered]
### Reproduction

https://codesandbox.io/s/vue-3-vuex-4-vue-router-forked-k9f3fh?file=/src/App.vue

### Steps to reproduce the bug

Open App and click to about link 

### Expected behavior

go to about page 

### Actual behavior

error: 
Unhandled error during execution of scheduler flush. This is likely a Vue internals bug. Please open an issue at https://new-issue.vuejs.org/?repo=vuejs/core 
  at <Products onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< null > > 
  at <RouterView> 
  at <App>

**Comment by Shyam-Chen** [comment, marked as answer]
`$route.params.id`
